app.dstack.yaml

```diff
@@ -0,0 +1,50 @@
+type: service
+
+name: aana-chat-with-video
+
+image: nvidia/cuda:12.3.2-cudnn9-devel-ubuntu22.04
+
+env:
+  - NUM_WORKERS=5
+  - TASK_QUEUE__EXECUTION_TIMEOUT=10000
+  - TASK_QUEUE__NUM_WORKERS=5
+  - TMP_DATA_DIR=/demo_data/aana
+  - IMAGE_DIR=/demo_data/aana/images
+  - VIDEO_DIR=/demo_data/aana/videos
+  - AUDIO_DIR=/demo_data/aana/audios
+  - MODEL_DIR=/demo_data/aana/models
+  - DB_CONFIG__DATASTORE_TYPE=sqlite
+  - DB_CONFIG__DATASTORE_CONFIG__PATH=/demo_data/aana.db
```

Comment:
I think the main disk is temporary so if the instance got restarted all the installation will be gone.
I am not sure if we need to consider this or not.

Reply:
`/demo_data` is mounted to a volume. So the data will persist.

```taml
volumes:
  - name: demo-data
    path: /demo_data
```

Reply:
I mean the repo itself. Like app files and also the virtual env of poetry

Reply:
Repo itself is cloned by dstack. If you restart the app through dstack it will clone it again. venv is always recreated in that case as a part of the startup sequence.